datasets/kwaliteitsmonitorwonen/dataset.json

```
@@ -0,0 +1,160 @@
+{
+  "id": "kwaliteitsmonitorwonen",
+  "type": "dataset",
+  "title": "Kwaliteitsmonitor Wonen",
+  "description": "Per verblijfsobject de kenmerken van de woningvoorraad van de gemeente Amsterdam.",
+  "version": "1.0.0",
+  "status": "beschikbaar",
+  "theme": [
+    "Wonen"
+  ],
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam",
+  "keywords": [
+    "Wonen"
+  ],
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "kwaliteitsmonitorwonen",
+      "type": "table",
+      "version": "1.0.0",
+      "title": "Kwaliteitsmonitor Wonen",
+      "schema": {
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "naam",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding objecttype"
+          },
+          "vbo_identificatie": {
```

Identifiers *moeten* in camelCase:

```suggestion
          "vboIdentificatie": {
```